scripts/spark2diffs/RapidsConf.diff

```diff
@@ -5,23 +5,23 @@
 23a24,25
 > import ai.rapids.cudf.Cuda
 > 
-321c323
+333c335
 <     .createWithDefault(ByteUnit.GiB.toBytes(1).toLong)
 ---
 >     .createWithDefault(ByteUnit.GiB.toBytes(1))
```

Comment:
these  shouldn't have changed, does this build?

Reply:
Same as previous comments. Github shows the lines around the changed line.